### ISSUE TITLE
problem: Update from radio button to slider interface reset all queue lengths to zero

### ISSUE THREAD

**Issue by jimkleine** [gazhayes/howlongistheline.org#56]
We need to be careful that interface updates don't discard useful information.

**Comment by gazhayes** [comment]
I'll push this to production tonight, then it won't matter because the lines will all have 0 people in them anyway.

**Comment by jimkleine** [comment]
Sorry I if I seemed a bit terse: I was on a phone to call to a family member out of Auckland and typing with the other hand.   I happened to have the site open and it refreshed with the new interface.

solution 1: Pre-seed the alternate data structure prior to the interface change (this isn't so critical that a handful of lost updates would be a significant problem).

solution 2: Roll out "breaking" changes after 9:00pm NZT (since most *NZ* stores are operating reduced hours and are closed by then).  NB There is one caveat to this: We currently have at least one international user (10 stores in Edmonton, Canada that I noticed earlier).  I'll open a "placeholder" issue to add a country field to the address details for future use.
